src/main/java/frc/robot/RobotContainer.java

```diff
@@ -51,8 +53,9 @@ public class RobotContainer {
   private LedStripSubsystem ledSubsystem;
   private LedMatrixSubsystem ledMatrixSubsystem;
   private BlinkinLedSubsystem blinkinLedSubsystem;
-
+  
   private Auton auton;
+  private TimeOfFlightSensor timeOfFlightSensor = new TimeOfFlightSensor();
```

Comment:
This should be instantiated only if tofEnabled == true.  

Leave it unassigned here, and create a new one when creating the subsystems.